### ISSUE TITLE
Support blank GOOGLE_NODE_RUN_SCRIPTS build_environment_variable

### ISSUE THREAD

**Issue by deriegle** [google-github-actions/deploy-cloud-functions#379]
### TL;DR

The Node.js buildpack was just recently updated to automatically run `npm run build`. You can disable this by setting the build environment variable of `GOOGLE_NODE_RUN_SCRIPTS` to a blank value. This doesn't seem possible today with how the environment variables are parsed.

[Relevant documentation](https://cloud.google.com/docs/buildpacks/nodejs#google_node_run_scripts)

### Expected behavior

I expected to be able to set the `GOOGLE_NODE_RUN_SCRIPTS` variable to a blank value within the `build_environment_variables` input.

### Observed behavior

You need to provide some value to the environment variable or it will fail to be parsed. If you pass a value, like `""` then you get the actual value of `""` rather than a blank value.

### Action YAML

```yaml
name: cd

on:
  push:
    branches: ['main']
  workflow_dispatch:

concurrency:
  group: ${{ github.workflow }}
  cancel-in-progress: false

jobs:
   deploy-function:
    runs-on: ubuntu-latest

    permissions:
      contents: 'read'
      id-token: 'write'

    steps:
      - uses: actions/checkout@v3
        with:
          fetch-depth: 2

      - id: 'auth'
        uses: 'google-github-actions/auth@v1'
        with:
          workload_identity_provider: 'projects/123456789/locations/global/workloadIdentityPools/my-pool/providers/my-provider'
          service_account: 'my-service-account@my-project.iam.gserviceaccount.com'
  
      - uses: google-github-actions/deploy-cloud-functions@v1
        with:
          name: myFunction
          runtime: nodejs18
          build_environment_variables: "GOOGLE_NODE_RUN_SCRIPTS="
```

### Log output

```text
Run google-github-actions/deploy-cloud-functions@v1
  with:
    name: myFunction
    runtime: nodejs18
    region: us-central1
    docker_registry: artifact-registry
    build_environment_variables: GOOGLE_NODE_RUN_SCRIPTS=
Error: google-github-actions/deploy-cloud-functions failed with: failed to parse KEY=VALUE pair "GOOGLE_NODE_RUN_SCRIPTS=": no value
```

### Additional information

I'd be happy to help out with this issue if needed.

**Comment by NickDub** [comment]
Any news about this issue?

**Comment by BoscoDomingo** [comment]
FYI you can also create a script in package.json called `gcp-build` and leave it blank. This will have the same effect as what you're looking for. [Source](https://cloud.google.com/functions/docs/concepts/nodejs-runtime#npm_build_script)

The issue you've mentioned is definitely a thing, but at least this could serve as a workaround if it's still an issue for you!

**Comment by BoscoDomingo** [comment]
> FYI you can also create a script in package.json called `gcp-build` and leave it blank. This will have the same effect as what you're looking for. [Source](https://cloud.google.com/functions/docs/concepts/nodejs-runtime#npm_build_script)
> 
> The issue you've mentioned is definitely a thing, but at least this could serve as a workaround if it's still an issue for you!

@NickDub @deriegle Sorry, forgot to mention you guys!
